paperqa/utils.py

```diff
@@ -52,10 +52,12 @@ def name_in_text(name: str, text: str) -> bool:
 def maybe_is_text(s: str, thresh: float = 2.5) -> bool:
     if not s:
         return False
+
     # Calculate the entropy of the string
     entropy = 0.0
+    s_wo_spaces = s.replace(" ", "")
```

Comment:
Can you update the comment two lines up to mention the heuristic for calculating entropy?

Reply:
I added a docstring with the reasoning for entropy calculation and leaving out spaces.